pkg/apis/solr/v1beta1/solrprometheusexporter_types.go

```diff
@@ -18,6 +18,7 @@ package v1beta1
 
 import (
 	"fmt"
+
```

Comment:
Sorry, I am using the VS Code editor. Maybe our editor linting is colliding a bit. Curious what do you use? I can either switch up of adjust the linter to make suggestions only perhaps.